src/efi/block.rs

```diff
@@ -106,11 +106,14 @@ pub struct BlockWrappers<'a> {
     pub count: usize,
 }
 
-pub extern "win64" fn reset(_: *mut BlockIoProtocol, _: bool) -> Status {
+eficall! {
```

Comment:
This looks a bit weird to me but I guess rustfmt is happy with this!

Reply:
Unfortunately, rustfmt is not happy with functions wrapped around macro because it cannot apply code format in the macro.